src/main/java/org/mqttbee/api/mqtt/mqtt5/Mqtt5ClientBuilder.java

```diff
@@ -65,26 +68,89 @@ public Mqtt5ClientBuilder(
     }
 
     @NotNull
-    public Mqtt5ClientBuilder followingRedirects(final boolean followsRedirects) {
+    @Override
+    public Mqtt5ClientBuilder identifier(@NotNull final String identifier) {
+        super.identifier(identifier);
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder identifier(@NotNull final MqttClientIdentifier identifier) {
+        super.identifier(identifier);
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder serverHost(@NotNull final String host) {
+        super.serverHost(host);
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder serverPort(final int port) {
+        super.serverPort(port);
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder useSsl() {
+        super.useSsl();
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder useSsl(@NotNull final MqttClientSslConfig sslConfig) {
+        super.useSsl(sslConfig);
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public Mqtt5ClientBuilder useWebSockets() {
```

Comment:
we should change it to "useWebsocket" as it only uses 1 not many

Reply:
DONE